guides/image-sources/provisioning-installer-httpboot.plantuml

```diff
@@ -0,0 +1,59 @@
+@startuml
+
+!include foreman.pstyle
+
+title Installer-based provisioning with UEFI HTTP boot on bare metal
+
+actor       User
+participant "Provisioned\nHost" as Host
+participant "Foreman" as Foreman
+participant "Foreman\nProxy" as Proxy
```

Comment:
```suggestion
participant "Foreman\nProxy\nHTTPboot" as Proxy
```
@stejskalleos Perhaps I could add this information to the Proxy participant so that it's clearer that it has to be a SmartProxy with the HTTPboot feature?

Reply:
+1